src/drivers/abstract/spellbook.js

```diff
@@ -476,12 +476,22 @@ function formatUpdate(spell) {
     for (const condition of whereConditions) collectLiteral(spell, condition, values);
     chunks.push(`WHERE ${formatConditions(spell, whereConditions)}`);
   }
+
+  const extraOptions = this.formatUpdateExtraOptions(spell);
+  if (extraOptions.length) {
+    chunks.push(...extraOptions);
+  }
```

Comment:
This bug fix shouldn't be sequelize adapter specific I think. Use cases like below should be supported as well:

```js
Post.update({ title: null }, { title: 'Untitled' }).order('id', 'desc').limit(10);
```

Reply:
It a MySQL only feature, this was supported at this PR, see 
* test/integration/suite/basics.test.js:685
* test/unit/spell.test.js:638

Reply:
> This bug fix shouldn't be sequelize adapter specific I think. Use cases like below should be supported as well:
> 
> ```js
> Post.update({ title: null }, { title: 'Untitled' }).order('id', 'desc').limit(10);
> ```
It's a MySQL only feature, this was supported at this PR, see

test/integration/suite/basics.test.js:685
test/unit/spell.test.js:638